zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsOptionsPanel.java

```diff
@@ -50,7 +51,11 @@
     private JCheckBox checkBoxConfirmDropMessage = null;
     private JCheckBox checkBoxAlwaysOnTop = null;
     private JCheckBox checkBoxInScopeOnly = null;
+    private JCheckBox checkBoxShowIgnoreFilesButtons = null;
```

Comment:
I wonder if this one would be better in the `Display` panel, this is closely related to "Show break buttons", we can then disable this if the option is set to "Only in the toolbar".

Reply:
I don't agree. I think that if the user only wants the break buttons on the toolbar it doesn't imply that he doesn't want to have the filter buttons on the break tab.

Reply:
When "Only in the toolbar" is selected no button is shown in the Break tab.

Reply:
Now when the  "Only in the toolbar" mode is on I deselect the option in the breakpoint options params.